pendulum/utils/_compat.py

```diff
@@ -6,8 +6,20 @@
 PY38 = sys.version_info[:2] >= (3, 8)
 
 if sys.version_info < (3, 9):
+    import importlib_resources as resources
+
     from backports import zoneinfo
 else:
     import zoneinfo
 
-__all__ = ["zoneinfo"]
+    from importlib import resources
+
+
+if sys.version_info < (3, 8):
+    from typing_extensions import Literal
```

Comment:
[Aliasing things like `Literal`](https://github.com/microsoft/pyright/issues/3025#issuecomment-1035266830) is not supported in pyright, so this one should probably be done inline where it is needed

Reply:
We are only supporting `mypy` check in our pipelines right now and I stick to how `mypy` works (never worked with `pyright`, but it sounds like it's not compatible with `mypy`).

Reply:
`pyright` and `mypy` work pretty hard to be compatible with one another. `pyright` does an early scan for some special forms to speed things up, and the `Literal` import is one of those. This is one of the few differences (aside from `pyright` supporting more PEPs) between `pyright` and `mypy`. The other thing to take into account is that `pyright` is the type checker that is included by default with Visual Studio Code's Python extension.